docs/content/docs/deployment/java_compatibility.md

```diff
@@ -50,7 +50,8 @@ The following Flink features have not been tested with Java 11:
 
 ## Java 17
 
-Experimental support for Java 17 was added in 1.18.0. ([FLINK-15736](https://issues.apache.org/jira/browse/FLINK-15736))
+Since version 1.19.0 Flink works reasonably well with Java 17, with the caveat that the integration is incomplete.
```

Comment:
I would keep the "Experimental support for Java 17 was added in 1.18.0. " line, as it gives more context.

if you just say "works reasonable well", it is not very convincing, and rather sounds like we don't know what we are doing here or what the status is.
Do we know what the caveats are?

Reply:
Fair enough. Caveats are listed in the `Known issues` section.